### PR TITLE
MET-5492 GDPR Consent Sandbox

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

**Review by jeortizquan — COMMENTED** [review]
I just have the observations in the comments.

File: projects/sandbox/src/app/cookie-consent/cookie-consent.component.scss [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+.eu-cm-ctrls {
+  display: flex;
+  flex-direction: column;
+  & > * {
+    margin: 1em 0;
+
+    color: $colorLink;
+    background-color: transparent;
+    border: 1px solid $colorTheme;
+
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/sandbox/src/app/cookie-consent/cookie-consent.component.scss:64]: I don't really like the uppercase on the buttons (it seems to shout) but I've added it for all buttons in the modal view and the last 2 in the mini-mode.

I think the "let me choose" should not be uppercase, as it's supposed to look like a link in the mini-mode to invite them into the modal.

File: projects/sandbox/src/environments/eu-cm-settings.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+};
+
+export const cookieConsentConfig = {
+  privacyPolicyUrl: getEnvVar('privacyPolicyUrl') as string,
+  services: [
+    {
+      name: 'matomo',
+      label: 'matomo tracker',
+      description: 'Collects anonymous statistics on how visitors interact with the website.',
+      purposes: ['usage tracking', 'tracking that'],
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/sandbox/src/environments/eu-cm-settings.ts]: I've removed one of the purposes and tweaked the other.  The text has not been finalised and I wanted to show an example of purposes.

File: projects/sandbox/src/app/cookie-consent/cookie-consent.component.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeortizquan) [reply projects/sandbox/src/app/cookie-consent/cookie-consent.component.scss:64]: afbeelding

**Review by jeortizquan — APPROVED** [review]

**Pull request merged**